File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2251.c
#include <stdio.h>
#include <math.h>

int main(){
    int tst[100001];
    int k=1;
    int i=0;
    while(k!=0){
        scanf("%d", &k);
        tst[i] = k;
        i++;
    }

    i=0;
    while(tst[i]!=0){
        tst[i] = pow(2,tst[i])-1;
        i++;
    }

    i=0;
    while(tst[i]!=0){
        printf("Teste %d\n%d\n\n", i+1, tst[i]);
        i++;
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2251.cs
using System;

class URI
{
    static void Main() {
        int i = 1;
        while(true) {
            int n = Int32.Parse(System.Console.ReadLine().Trim());
            if(n == 0) break;
            
            Console.Write("Teste {0}\n", i);
            Console.Write("{0:f0}\n", Math.Pow(2, n)-1);
            Console.Write("\n");
            
            i++;
            
        }
    }
}
